libspawner/src/posix/securerunner.cpp

```diff
@@ -93,7 +93,7 @@ bool secure_runner::create_restrictions() {
     if (check_restriction(restriction_memory_limit))
         // linux&cygwin both supports Address Space rlimit
 #if defined(__linux__) || defined(__CYGWIN__)
-        impose_rlimit(RLIMIT_AS, restrictions.get_restriction(restriction_memory_limit));
+        impose_rlimit(RLIMIT_AS, 2*restrictions.get_restriction(restriction_memory_limit));
```

Comment:
Cygwin does not have procfs support, so there is no need to double address space limit in this case.
Please discuss Cygwin suppor for the project t w/ Aleksander Sergeevich.